camera.c

```diff
@@ -11,16 +11,16 @@ int getName(char name[]);// to be used in openPic to check if the name is in the
 int main(){
   char name[10];
   //greeting and menu options
-  char opt;
+  char *opt;
```

Comment:
this is causing the segmentation fault(I think), I'm trying to access memory I don't have access to.